k2/csrc/weights.h

```diff
@@ -18,7 +19,11 @@
 
 namespace k2 {
 
-constexpr float kNegativeInfinity = -std::numeric_limits<float>::infinity();
+constexpr float kFloatNegativeInfinity =
+    -std::numeric_limits<float>::infinity();
+constexpr float kDoubleNegativeInfinity =
+    -std::numeric_limits<double>::infinity();
+constexpr double kShortestDelta = 1e-6;
```

Comment:
Is this variable unused?

Reply:
I deleted code that do `ApproxEqual` in `Wfsa`, will delete it